src/main/java/fmartin1/controller/PokedexController.java

```diff
@@ -0,0 +1,79 @@
+package fmartin1.controller;
+
+
+import fmartin1.model.pokemon.Pokemon;
+import fmartin1.model.pokemon.PokemonComparator;
+import fmartin1.service.PokedexService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.stream.Collectors;
+
+@SuppressWarnings("unused")
+@Controller
+@RequestMapping("pokemon")
+public class PokedexController {
+    private final PokedexService _pokedexService;
+
+    @Autowired
+    public PokedexController(PokedexService pokedexService) {
+        _pokedexService = pokedexService;
+    }
+
+    @RequestMapping(value = "/", method = RequestMethod.GET, produces = MediaType.TEXT_PLAIN_VALUE)
```

Comment:
Update this method to return a JSON Object, you can use `gSON` or `Jackson` to do this